kasa/smart/smartmodule.py

```diff
@@ -57,7 +57,7 @@ class SmartModule(Module):
     #: Module is initialized, if any of the given keys exists in the sysinfo
     SYSINFO_LOOKUP_KEYS: list[str] = []
     #: Query to execute during the main update cycle
-    QUERY_GETTER_NAME: str
+    QUERY_GETTER_NAME: str = ""
```

Comment:
Maybe this belongs into its own PR?

Reply:
I think that as it's the `smartcam` `homekit` module that has the use case for this and it's so small that it's good to keep in this PR.

Reply:
As discussed on discord we're good to keep this here